source/CardGames.py

```diff
@@ -85,6 +85,17 @@ def getCard( suit, value):
       return card
   return None
 
+####################################################################################
+def count_cards(deck):
```

Comment:
Your function looks good. However, it seems like there is room left for error handling. Is there any possibility the function could be imputed with something other than a valid deck object?

Reply:
well it only calls deck, so there shouldent beb any issue of anything else being pushed through 👍

Reply:
Got it. Great work!